spec/features/user_deletes_piece_spec.rb

```diff
@@ -0,0 +1,34 @@
+feature "user deletes a piece" do
+  let(:user) { FactoryGirl.create(:user) }
+  let!(:piece) { FactoryGirl.create(:piece, user: user) }
+
+  context "signed-in user cannot delete a piece" do
+    let(:user2) { FactoryGirl.create(:user) }
+
+    scenario "signed in user cannot see delete button for piece he did not create" do
```

Comment:
Another good scenario (pardon the bad pun) for avoiding line-length warning. This is fine!